espnet2/asr_transducer/decoder/modules/rwkv/attention.py

```diff
@@ -83,7 +84,7 @@ def backward(
 
         """
         time_decay, time_first, key, value, output = ctx.saved_tensors
-        grad_dtype = ctx.input_dtype
+        # grad_dtype = ctx.input_dtype
```

Comment:
Is it OK?

Reply:
The variable is assigned, but never used.